src/main/java/donnafin/logic/commands/AddCommand.java

```diff
@@ -52,7 +52,8 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         model.addPerson(toAdd);
-        return new CommandResult(String.format(MESSAGE_SUCCESS, toAdd));
+        String personString = toAdd.toString().replace("Notes:", "");
```

Comment:
Would it be better to leave all of these files unchanged and update `Person.toString()` to not append `+ "Notes: " + notes` when `notes.equals("")`?